website/content/api-docs/secret/cubbyhole.mdx

```diff
@@ -105,7 +105,7 @@ This endpoint stores a secret at the specified location.
 - `path` `(string: <required>)` – Specifies the path of the secrets to
   create/update. This is specified as part of the URL.
 
-- `:key` `(string: "")` – Specifies a key, paired with an associated value, to
+- `:key` `(string: "")` – Specifies a key in the payload, paired with an associated value, to
```

Comment:
I made this change in both cubbyhole and KVV1's docs, as we don't normally show payload keys this way, and I thought it would benefit from the addition.